deployment/environments/dev/values.yaml

```diff
@@ -1,6 +1,18 @@
-contracts: 0xD0b57307a4B9e0147F389ab99547E1530D323be0
-rpc: https://polygon-mumbai-bor.publicnode.com
-debug: true
-startingBlock: "41924130"
-blocksMargin: 70
-blocksRange: 2
+networks:
+  - name: polygon-mumbai
+    contracts: 0xD0b57307a4B9e0147F389ab99547E1530D323be0
+    rpc: https://polygon-mumbai-bor.publicnode.com
```

Comment:
can this be in secrets (or config) so that we can change L1 if this does not work?

Reply:
since we're using a public node it does not make much sense to put it in the secrets. If you want to change it, it's easier to change it in the deployment

Reply:
but you can change it without modifying code... it's the same only in the secret you have it encrypted